.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.github
```

Comment:
- [ ] I noticed that you added the `.github` directory to your `.gitignore` file. Please note that the `.github` directory is where linter files are typically stored.
To ensure that your linter files are included in your repository, I suggest removing the `.github` directory from your `.gitignore` file. This will help ensure that your project is properly configured and that your linter files are accessible to other developers who may want to use or contribute to your project. 🎯